templates/instruc_template.forTasks.rst

```diff
@@ -0,0 +1,236 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+########################
+Guidance for ExampleTask
```

Comment:
I recommend a more useful title:

```
Writing Task Topics
```

I both prefer this grammar, and also, `ExampleTask` has no real meaning.